README.md

```diff
@@ -4,8 +4,27 @@
 
 1. Install Ruby `3.2.1`.
 2. Create `.env` file in root folder of the project.
-2. Install [PostgreSQL](https://www.postgresql.org/download/) `>=14.13`.
-3. Add corresponded env vars to `.env` file with DB credentials. E.g.:
+3. Install [PostgreSQL](https://www.postgresql.org/download/) `>=14.13`.
+   '''
+   sudo apt update
```

Comment:
This [works only](https://en.wikipedia.org/wiki/APT_(software)) on Debian-based Linux distributions like Ubuntu, Linux Mint, and Debian itself.
I suggest to either add other options (e.g. for Mac/Windows) or just point that this is only for Debian-based distributions.